parser/mpFuncNonCmplx.cpp

```diff
@@ -100,8 +119,9 @@ double round(long_double_type number, int_type precision) {
     MUP_UNARY_FUNC(FunCbrt,  "cbrt",  std::cbrt,  "cbrt(x) - cubic root of x")
     MUP_UNARY_FUNC(FunExp,   "exp",   std::exp,   "exp(x) - e to the power of x")
     // number functions
-    MUP_UNARY_FUNC(FunAbs,   "abs",   std::fabs,  "abs(x) - absolute value of x")
-    MUP_UNARY_FUNC(FunRound, "round", std::round, "round(x) - round the value of x to its nearest integer")
+    MUP_UNARY_FUNC(FunAbs,   "abs",    std::fabs,  "abs(x) - absolute value of x")
+    MUP_UNARY_FUNC(FunRound, "round",  std::round, "round(x) - round the value of x to its nearest integer")
+    MUP_UNARY_FUNC(FunString,"string", to_string,  "string(x) - converts the number from decimal to string")

```

Comment:
Hmmmm! Interesting this mapping

Reply:
This `#define MUP_UNARY_FUNC` is amaaazing! I just understood it now!!